canonical-web-and-design/jaas.ai
# Copy the global-nav module to the project

## Done
Copy the global-nav module to the project to share modules across discourse and other juju sites.

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029/static/js/modules/global-nav/global-nav.js
- Check you see the matrix of js.
